src/auth/axios.ts

```diff
@@ -51,7 +59,9 @@ const responseErrorInterceptor = (error: AxiosError) => {
     error?.response?.data === INVALID_ACCESS_TOKEN &&
     !isTokenValid(tokens.result.refreshToken)
   ) {
-    logout()(store.dispatch, store.getState, userAuthenticationExtraArgs);
+    await logout()(store.dispatch, store.getState, userAuthenticationExtraArgs);
+    localStorage.removeItem(LOCALSTORAGE_STATE_KEY);
```

Comment:
📝 Honestly not entirely sure why we have to clear the storage again - I assume it has something to do with the context.

Reply:
If on a public computer then its possible to grab the token in a XSS and login to your account.